Integration test issue

1. Invalid email
```
E               dohq_artifactory.exception.ArtifactoryException: {
E                 "errors" : [ {
E                   "status" : 400,
E                   "message" : "invalid email user1"
E                 } ]
E               }
```
2. Invalid password
```
E               dohq_artifactory.exception.ArtifactoryException: {
E                 "errors" : [ {
E                   "status" : 400,
E                   "message" : "New password doesn't qualify password policy: digit requirement is invalid. Should include at least 1"
E                 } ]
E               }
```
3. Assert list
```
E       AssertionError: assert ['admin'] is ['admin']
E        +  where ['admin'] = <Group test_adding_user_to_group>.users
```
4. Warning
```
tests/integration/test_integration_artifactory_path.py::test_read_and_write
  /Users/xxx/.pyenv/versions/3.9.5/envs/v-arti/lib/python3.9/site-packages/charset_normalizer/api.py:105: UserWarning: Trying to detect encoding from a tiny portion of (16) byte(s).
    warn('Trying to detect encoding from a tiny portion of ({}) byte(s).'.format(length))
```

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -137,7 +137,7 @@ def user1(artifactory):
     if user is not None:
         user.delete()
     user = User(
-        artifactory=artifactory, name=name, email=name, password=generate_password()
+        artifactory=artifactory, name=name, email=f"{name}@example.com", password="Pa55w@rd"
     )
     user.create()
     yield user
@@ -151,7 +151,7 @@ def user2(artifactory):
     if user is not None:
         user.delete()
     user = User(
-        artifactory=artifactory, name=name, email=name, password=generate_password()
+        artifactory=artifactory, name=name, email=f"{name}@example.com", password="Pa55w@rd"
     )
     user.create()
     yield user

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -5,7 +5,6 @@ import sys
 import pytest
 
 from artifactory import ArtifactoryPath
-from dohq_artifactory import generate_password
 from dohq_artifactory import Group
 from dohq_artifactory import PermissionTarget
 from dohq_artifactory import RepositoryLocal
@@ -137,7 +136,10 @@ def user1(artifactory):
     if user is not None:
         user.delete()
     user = User(
-        artifactory=artifactory, name=name, email=f"{name}@example.com", password="Pa55w@rd"
+        artifactory=artifactory,
+        name=name,
+        email=f"{name}@example.com",
+        password="Pa55w@rd",
     )
     user.create()
     yield user
@@ -151,7 +153,10 @@ def user2(artifactory):
     if user is not None:
         user.delete()
     user = User(
-        artifactory=artifactory, name=name, email=f"{name}@example.com", password="Pa55w@rd"
+        artifactory=artifactory,
+        name=name,
+        email=f"{name}@example.com",
+        password="Pa55w@rd",
     )
     user.create()
     yield user

--- a/tests/integration/test_admin.py
+++ b/tests/integration/test_admin.py
@@ -18,7 +18,7 @@ class TestUser:
             artifactory=artifactory,
             name=user_name,
             email="test_user@example.com",
-            password="password",
+            password="Pa55w@rd",
             profile_updatable=True,
         )
 
@@ -42,7 +42,7 @@ class TestUser:
             artifactory=artifactory,
             name=user_name,
             email="test_user@example.com",
-            password="oldpassword",
+            password="oldPa55w@rd",
         )
 
         # CREATE
@@ -51,9 +51,9 @@ class TestUser:
 
         # UPDATE
         test_user = artifactory.find_user(user_name)  # type: User
-        test_user.password = "oldpassword"
+        test_user.password = "oldPa55w@rd"
         current_pwd = test_user.encryptedPassword
-        test_user.password = "newpassword"
+        test_user.password = "newPa55w@rd"
         test_user.update()
         new_pwd = test_user.encryptedPassword
 
@@ -109,7 +109,7 @@ class TestGroup:
         assert test_group is not None
 
         test_group.read()
-        assert test_group.users is users
+        assert test_group.users == users
 
         # DELETE
         test_group.delete()

--- a/tests/integration/test_integration_artifactory_path.py
+++ b/tests/integration/test_integration_artifactory_path.py
@@ -191,8 +191,10 @@ def test_read_and_write(path):
     if p.exists():
         p.rmdir()
 
-    p.write_text("Some test string")
-    assert p.read_bytes() == b"Some test string"
-    assert p.read_text() == "Some test string"
+    # If the length of sequence is less than 32, there will be a warning msg.
+    # UserWarning: Trying to detect encoding from a tiny portion of (16) byte(s).
+    p.write_text("Some test string ensure length > 32")
+    assert p.read_bytes() == b"Some test string ensure length > 32"
+    assert p.read_text() == "Some test string ensure length > 32"
 
     p.unlink()

